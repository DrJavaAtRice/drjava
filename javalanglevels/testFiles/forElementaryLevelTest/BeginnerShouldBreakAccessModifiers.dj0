/*!DrJava Beginner Level*/

class BeginnerShouldBreakAccessModifiers extends Object{
  public int i;
  private int j;
  protected char c;
}