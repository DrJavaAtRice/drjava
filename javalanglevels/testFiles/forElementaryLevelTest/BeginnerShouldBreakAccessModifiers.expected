// Language Level Converter line number map: dj*->java. Entries: 7
//     1->3         2->4         3->5         4->6         5->7         6->8         7->9    
/*!DrJava Beginner Level*/

class BeginnerShouldBreakAccessModifiers extends Object{
  final public int i;
  final private int j;
  final protected char c;

  /** This method is automatically generated by the Language Level Converter. */
  public BeginnerShouldBreakAccessModifiers(int i, int j, char c) {
    super();
    this.i = i;
    this.j = j;
    this.c = c;
  }

  /** This method is automatically generated by the Language Level Converter. */
  public int i() {
    return i;
  }

  /** This method is automatically generated by the Language Level Converter. */
  public int j() {
    return j;
  }

  /** This method is automatically generated by the Language Level Converter. */
  public char c() {
    return c;
  }

  /** This method is automatically generated by the Language Level Converter. */
  public java.lang.String toString() {
    return getClass().getName() + "(" + 
        i() + ", " + 
        j() + ", " + 
        c() + 
        ")";
  }

  /** This method is automatically generated by the Language Level Converter. */
  public boolean equals(java.lang.Object o) {
    if (this == o) {
      return true;
    }
    else if ((o == null) || (! o.getClass().equals(getClass()))) {
      return false;
    }
    else {
      BeginnerShouldBreakAccessModifiers cast = ((BeginnerShouldBreakAccessModifiers) o);
      return (i() == cast.i()) && 
          (j() == cast.j()) && 
          (c() == cast.c());
    }
  }

  /** This method is automatically generated by the Language Level Converter. */
  public int hashCode() {
    return getClass().hashCode() ^ 
        i() ^ 
        j() ^ 
        c();
  }
}
